runners/flink/runner/pom.xml

```diff
@@ -178,10 +170,71 @@
       <plugin>
         <groupId>org.apache.maven.plugins</groupId>
         <artifactId>maven-surefire-plugin</artifactId>
+        <executions>
```

Comment:
@davorbonaci @jasonkuster @lukecwik my maven-fu might be lacking. I think I can remove the `phase` and `goals` here, since the execution has an id, and maybe lift some of the configuration, but most importantly, I could not get `runnableOnServicePipelineOptions` to do anything, instead having to set `beamTestPipelineOptions`. This seems mostly fine but please advise.